components/ShadowLayer/src/MDCShadowLayer.m

```diff
@@ -15,8 +15,8 @@
 #import "MDCShadowLayer.h"
 
 static const CGFloat kShadowElevationDialog = 24.0;
-static const float kKeyShadowOpacity = 0.26f;
-static const float kAmbientShadowOpacity = 0.08f;
+static const float kKeyShadowOpacity = (float)0.26;
```

Comment:
are these conversions needed?

Reply:
Yep, those values are typed to `float`. I didn't want to switch them to CGFloat since I believe the properties themselves are `float`.